CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+
+Most of the links above will directly open an issue for you to fill out a description of the problem or request, but several of the repositories provide choices (and related description templates) for the type of issue you want to open. Those choices could include the following:  
+
+* **Feature Request** or **Course Template Feature Request**: Use this issue if you want to make a suggestion for an addition.
+* **Problem Report** or **Course Template Problem Report**: Use this issue if you want to report a problem or possible improvement.
+* **New Content Idea**: This is a template meant for opening an issue in an OTTR course, so that someone can suggest an additional topic or idea for that specific course. It shouldn't be used for opening an issue in one of these repos.
+* **Course Problem Report**: This is a template meant for opening an issue in an OTTR course, so that someone can point out a problem or suggest an improvement for that specific course. It shouldn't be used for opening an issue in one of these repos.
+
+##### Assign an OTTR maintainer
+
+Assign an OTTR maintainer (`@cansavvy` or `@carriewright11`) to the issue.
+
+#### Google Form
+
+This [OTTR Feedback Google Form](https://docs.google.com/forms/d/e/1FAIpQLSfcc_h1MPfCOWyQup4Z76lzlBGz8cAkDQmBiqfxqOZ4_4lCkQ/viewform) can be used to ask questions about any problems you encounter or submit feedback such as ideas or requests for improvements.
+
+### You want to make suggestions, and implement them yourself!
+
+If you want to implement an improvement or fix a problem, open a branch, make the changes to the code there, and then open a pull request. [See this chapter "Using Version Control with GitHub"](https://jhudatascience.org/Adv_Reproducibility_in_Cancer_Informatics/using-version-control-with-github.html) from our Advanced Reproducibility course for more information on this pull request model if needed.  
+
+#### Request Review
+
+Make sure to request review of your pull request from an OTTR maintainer (`@cansavvy` or `@carriewright11`).
+
+## What if you want to make several courses with similar branding using OTTR?
+
+If you plan to use the OTTR template several times for related courses within an organization and want each to have a shared branding or functionality variation, you may want to make your own template from the OTTR_Template, incorporating the stylistic changes in your spinoff template before making specific courses from the spinoff template.
+
+OTTR_Template --> Spinoff_Template (Make stylistic changes, mark as a template repository) --> Course Repository
+
+Examples that have done this include:
+
+| Spinoff template   | Link |
+|:------------------:|:-------------------------------------------------------------------------:|
+| AnVIL Template     | [GitHub Source Code](https://github.com/jhudsl/AnVIL_Template)            |
+| DataTrail Template | [GitHub Source Code](https://github.com/datatrail-jhu/DataTrail_Template) |
```

Comment:
```suggestion
| DataTrail Template | [GitHub Source Code](https://github.com/datatrail-jhu/DataTrail_Template) |
|Quarto website| [GitHub Source Code](https://github.com/fhdsl/OTTR_Quarto)| 
|Metric collection dashboard template| [GitHub Source Code](https://github.com/fhdsl/metricminer-dashboard)|
```

These other templates are literally brand new so probably didn't exist when you started writing this. No worries. Adding them now!

Reply:
I only just added them to ottrproject.org yesterday! lol